empress/fileInput.py

```diff
@@ -0,0 +1,220 @@
+# ReconInput class
+from io import StringIO
+
+# BioPython libraries
+from Bio import Phylo
+
+class ReconInput(object):
+    """
+    Storage class for the newick data (trees, tip mapping, and optional distance parameters)
+    Distances encode branch lengths in the newick file, if given
+    """
+
+    def __init__(self, host_tree = None, host_distances = None, parasite_tree = None, 
+                parasite_distances = None, phi = None):
+        self.host_tree = host_tree
+        self.host_distances = host_distances
+        self.parasite_tree = parasite_tree
+        self.parasite_distances = parasite_distances
+        self.phi = phi
+
+    def complete(self, host_file: str, parasite_file: str, mapping_file = None):
```

Comment:
Maybe rename the function to `read(...)`.

Reply:
I don't know this is just what Prof Ran told me to name it.